src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/assembler/barrel.json

```diff
@@ -12,10 +12,16 @@
       "amount": 2
     }
   ],
+  "fluid_inputs": [
+        {
+            "fluid": "modern_industrialization:creosote",
```

Comment:
Not sure about using creosote as it's mean only for early game recipe ... maybe use lubricant instead

Reply:
Creosote was chosen because it is used irl to treat wood so it is more resilient to termites etc.  That, and I figured that since the people who are most likely to use these recipes are builders, they might still be using Coke Ovens for Coke Blocks and/or Dye production, and have excess Creosote lying around.